scripts/ci/cmake/ci-ubuntu20.04-gcc10-mpich.cmake

```diff
@@ -39,8 +39,17 @@ CMAKE_Fortran_FLAGS:STRING=-Wall
 MPIEXEC_MAX_NUMPROCS:STRING=${N2CPUS}
 ")
 
-set(CTEST_TEST_ARGS
-  EXCLUDE "Engine.BPEngineTest.SzComplex.MPI|Engine.BPEngineTest.ZfpComplex.MPI|.Serial$")
+# TODO: The Kill* and PreciousTimeStep tests fail (due to timeout) when
+# TODO: adios2 is built "--with-device=ch3:sock:tcp".  Once this is fixed
+# TODO:  in the mpi_dp, we can re-enable these tests.
+list(APPEND EXCLUDE_EXPRESSIONS
+  "Engine.BPEngineTest.SzComplex.MPI"
+  "Engine.BPEngineTest.ZfpComplex.MPI"
+  "KillReader"
+  "KillWriter"
+  "PreciousTimeStep")
```

Comment:
I guess this approach to excluding tests may not be working, since I can see those tests were run and failed [here](https://open.cdash.org/viewTest.php?onlyfailed&buildid=9150653).